docker/compose-tracing-v2-extra-extension.yml

```diff
@@ -0,0 +1,29 @@
+include:
+  - compose-tracing-v2.yml
+
+services:
+  shomei-frontend:
+    extends:
+      file: compose-spec-l2-services.yml
+      service: shomei-frontend
+
+  l2-node:
+    extends:
+      file: compose-spec-l2-services.yml
+      service: l2-node
+
+  l2-node-besu:
+    extends:
+      file: compose-spec-l2-services.yml
+      service: l2-node-besu
+
+  postman:
+    extends:
+      file: compose-spec-l2-services.yml
+      service: postman
+
+  transaction-exclusion-api:
+    extends:
+      file: compose-spec-l2-services.yml
+      service: transaction-exclusion-api
+
```

Comment:
This is to let user to run the additional services: l2-node, shomei-frontend, etc.. in local stack